### ISSUE TITLE
viewDate function does not notify component

### ISSUE THREAD

**Issue by vin-car** [tempusdominus/core#13]
Calling `viewDate` function with a valid moment object as argument does not notify changes to the view. Something like `$('#datetimepicker1').datetimepicker('viewDate', moment('11/21/2018', 'MM/DD/YYYY') );` has no effect on the datetimepicker.

Live example on [this question](https://stackoverflow.com/q/53421566/4131048) on Stack Overflow using jQuery 2.1.1, moment 2.22.2, bootstrap 4.1.3 and tempus dominus 5.1.2.

---

I think that the issue is the `_notifyEvent` function that always returns without trigger event since:

```js
if ((e.type === DateTimePicker.Event.CHANGE && (e.date && e.date.isSame(e.oldDate)) || !e.date && !e.oldDate))
```

is always true when [`this._viewUpdate()`](https://github.com/tempusdominus/core/blob/5.0.3/src/js/tempusdominus-core.js#L1486) is called from [`viewDate`](https://github.com/tempusdominus/core/blob/master/src/js/tempusdominus-core.js#L1471) function.

**Comment by lyhung009** [comment]
Any update on this issue.

**Comment by Fabjooo** [comment]
I have the same problem :/

**Comment by Eonasdan** [comment]
fixed via #31
